### ISSUE TITLE
Integrate with Searchkit

### ISSUE THREAD

**Issue by rrjanbiah** [meilisearch/integration-guides#60]
https://github.com/searchkit/searchkit by @searchkit is a ready to use React UI for Elastic search. I'm not sure if Meilisearch endpoints and responses are compatible with the searchkit. If not, may refer similar or other UI kits too in the integration guides.

**Comment by rrjanbiah** [comment]
@curquiza Yes, I have noted that https://github.com/meilisearch/meilisearch-react/ is the best go to kit at this moment. I thought that integrating with searchkit using some JSON transpose libraries (such as https://github.com/jmespath/jmespath.js, https://github.com/jsonata-js/jsonata or https://github.com/wankdanker/node-object-mapper) would be easier. But, I have also noted that they broke their UI in the recent version.

**Comment by curquiza** [comment]
These JSON libraries could be a good start, but not enough I think because the Elastic API and the MeiliSearch API are really different. The whole point of this kind of job is to make the components (designed for the Elastic API and configuration) work with the MeiliSearch configuration. We have to "connect" the equivalent configurations from Elastic with MeiliSearch. That's what we did with [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch) (so for meilisearch-react) and InsatntSearch components that are originally designed for Algolia configuration.

As I said, this is not impossible, but still a huge work. Bigger than the work we did with instant-meilisearch because Algolia and MeiliSearch APIs are quite close 🙂
